code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -111,3 +111,35 @@ Lizard subspecies: ASHWALKERS
 	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,DIGITIGRADE,HAS_FLESH,HAS_BONE)
 	inherent_traits = list(TRAIT_ADVANCEDTOOLUSER,TRAIT_CHUNKYFINGERS,TRAIT_NOBREATH)
 	species_language_holder = /datum/language_holder/lizard/ash
+
+/*
+Lizard subspecies: SILVER SCALED
+*/
+/datum/species/lizard/silverscale
```

Comment:
i think they should have natural armor like golems with their silver scales, since the other pirates have armored clothing but these guys dont

Reply:
They do, it's just a tad light.

Reply:
also limit changesource_flags to MIRROR_BADMIN | WABBAJACK i think